Pancake_Sort/PancakeSorting.java

```diff
@@ -0,0 +1,57 @@
+// Input n=5 array={5,4,3,2,1}
+import java.util.*;
+class Main
+{   
+    static void reverseArray(int a[],int n) //This functions reverses the array upto index n
+    {
+        int temp[]=new int[n+1]; // temp is the temporary array 
+        for(int i=0;i<=n;i++)
+        {
+            temp[i]=a[n-i];
+        }
+        for(int i=0;i<=n;i++)
+        a[i]=temp[i];
+     }

```

Comment:
provide space around all operators throughout the file

Reply:
By this do you mean to change 
for(i=0;i<=n;i++)
to 
for ( i = 0 ; i <= n ; i ++)